index.html

```diff
@@ -1,4 +1,5 @@
 <!DOCTYPE html>
+<!-- this is a change so I can give you a pull request -->
```

Comment:
We will be discussing pull requests in class. This is yours.  Most people have corrections to their code in their pull requests, but yours is fine as it is.  Well done!